conf.sh

```diff
@@ -14,14 +14,25 @@ if [ $1 ]; then
 	if [ $1 = "--static" ]; then
 		echo "configure to static compile!"
 		COMPILE_TYPE="--enable-static --disable-shared --with-static_drv"
+	elif [ $1 = "--with-uadk_v1" ]; then
+		echo "enable UADK v1!"
+		UADK_VERSION="--with-uadk_v1"
 	else
 		echo "invalid paramter, --static is static compile, compile to shared lib by default"
 	fi
 fi
 
+if [ $2 ]; then
+	if [ $2 = "--with-uadk_v1" ]; then
+		echo "enable UADK v1!"
+		UADK_VERSION="--with-uadk_v1"
+	fi
+fi
+
```

Comment:
confused here. Could we add some comments about the v1/v2, static/shared lib conf.sh command?

Reply:
OK. I'll update it with comments.